webapp/i18n/en.json

```diff
@@ -540,6 +540,24 @@
   "admin.saml.usernameAttrTitle": "Username Attribute:",
   "admin.saml.verifyDescription": "When true, Mattermost verifies that the signature sent from the SAML Response matches the Service Provider Login URL",
   "admin.saml.verifyTitle": "Verify Signature:",
+  "admin.cluster.loadedFrom": "This config file was loaded from Node ID {clusterId}.  If you are accessing the System Console through a load balancer you may experience issues.",
+  "admin.cluster.should_not_change": "WARNING: these settings may not sync with the other servers in the cluster.  You must verify the config files are identical and restart the cluster.  We will attempt to save the settings across the servers.  Changing from disabled to enabled will not start the High Availability inter-node communication and you will need to modify the config.json on the other servers and restart Mattermost.  Please see the documentation on how to enable and join or remove a server to the cluster. If you are accessing the System Console through a load balancer you may experience issues.  Yoy may need to access the admin console for each server in the cluster by accessing the internal/private ip address of the server.",
```

Comment:
Some of these sentences seem redundant. Please review for accuracy but I propose:

`WARNING: These settings may not sync with the other servers in the cluster. High Availability inter-node communication will not start until you modify the config.json to be identical on all servers and restart Mattermost. The System Console for each server in the cluster is accessible by the internal/private IP address.  Please see the <a href=\"http://docs.mattermost.com/deployment/cluster.html\" target=\"_blank\">documentation</a> on how to add or remove a server from the cluster. If you are accessing the System Console through a load balancer and experiencing issues, please see the Troubleshooting Guide in our <a href=\"http://docs.mattermost.com/deployment/cluster.html\" target=\"_blank\">documentation</a>. `

Also, do the IT admins need to access the system console for each server individually or can they make all the necessary modifications via the config files? If they can do it from the config files I'd propose removing the sentence `The System Console for each server in the cluster is accessible by the internal/private IP address.`